client/constants/Messages.mjs

```diff
@@ -0,0 +1,8 @@
+const MESSAGES = {
+  "no-energy": "You are exhausted, rest for a while",
+  "not-in-range": "You are too far away, come closer",
+  "no-resources": "Not enough resources",
+  "no-skill": "You are not skilled enough",
```

Comment:
There is no need for object keys names to be with dashes. More, it doesn't give us anything because you can't use it to read the property from an object:
`obj.key-name`
You can use lodash to work as property identifier, eg.
`obj.key_name` 
Treat is as a string identifier, it can be just a string
eg. `"no energy"`